data_dir/scylla-dash-per-server-nemesis.master.json

```diff
@@ -6792,7 +6792,7 @@
       ]
     },
     "timezone": "utc",
-    "title": "[longevity-twcs-3h] Scylla Per Server Metrics Nemesis Master",
+    "title": "[$test_name] Scylla Per Server Metrics Nemesis Master",
     "uid": "J1T2FZ57za",
```

Comment:
@dkropachev  what is `uid` here? May it be random string?

Reply:
It is used to identify the dash board in case you want to update it.